app/assets/stylesheets/trips.css.scss

```diff
@@ -1,3 +1,12 @@
+#map{
+  width: 70%;
```

Comment:
Properties should be ordered border, border-radius, height, margin-left, margin-right, width